skimage/measure/_regionprops.py

```diff
@@ -267,12 +267,12 @@ def __getattr__(self, attr):
                 f"'{type(self)}' object has no attribute '{attr}'"
             )
 
-    @property
+    @property # type: ignore
```

Comment:
Why is the type ignored here? In this example `np.sum` seems annoying to properly type annotated for all cases, but that is the problem of the numpy project. In the specific case of a property that just wraps `np.sum(self.image)` I would expect the return type to be simple float or `np.ndarray` or at least just `Union[float,np.ndarray]`, which is already a lot more information than type ignore. With `type: ignore` I had to lookup the type signature of `np.sum` myself now to even know that this can actually return an array and not just a float.

The same idea probably applies to the other `type: ignore` annotations in the rest of the file.

Reply:
See https://github.com/python/mypy/issues/1362, which has a lot of people complaining that mypy does not support the property decorator :-).

Reply:
You had me confused there for a minute and questioning my memories ;)
The issue is about a subtly more specific case: mypy doesn't support **decorated properties** but it supports the **property decorator**. So this _specific_ instance of `# type: ignore` might  actually be needed. But I am not even sure if this issue applies for just "properties that are decorated further" or also for "some function that is decorated and then wrapped with a property".
At least some of the cases further down only use `@property` though, and I am fairly sure that it should work in that case by simply annotating the return type of the decorated method.

Reply:
@fmagin we flagged this above in [this comment](https://github.com/scikit-image/scikit-image/pull/4794#discussion_r443397188), which admittedly GitHub makes difficult to see, and the reason is found here:

https://github.com/python/mypy/issues/1362